composites/Plugin/SnippetPreview/components/SnippetPreview.js

```diff
@@ -516,11 +527,19 @@ export default class SnippetPreview extends PureComponent {
 		if ( this.props.mode === MODE_MOBILE ) {
 			urlContent = this.getBreadcrumbs( cleanUrl );
 		} else {
-			urlContent = highlightKeyword( locale, keyword, url, cleanUrl );
+			/*
```

Comment:
comment block start is off by 1 space

Reply:
My bad, it was the other way around. All the other comment stars + the end should be indented with 1 space, the first line was fine.